msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md

```diff
@@ -1,31 +1,38 @@
 ---
-title: Add custom bots to Microsoft Teams with outgoing webhooks
-description: describes how to add an outgoing webhook
+
+title: Create an outgoing webhook
+description: describes how to create an outgoing webhook
 ms.topic: conceptual
-ms.author: lajanuar
 localization_priority: Normal
+ms.author: lajanuar
 keywords: teams tabs outgoing webhook actionable message verify webhook
+
 ---
-# Add custom bots to Teams with outgoing webhooks
 
```

Comment:
Global: Self review is not performed for this file. Please revisit and apply the comments given in the previous file.